kuberender/render.py

```diff
@@ -102,7 +102,21 @@ def apply_templates(rendered_templates):
         call_kubectl_apply(t)
 
 
-def run(verbose=False, template_dir='templates', should_apply=False, context_files=None, overriden_vars=None, template_url=None, working_dir='.'):
+def save_template(template, generated_dir):
+    path = '%s/%s' % (generated_dir, template.slug)
+    with open(path, 'w') as f:
+        f.write(template.content)
+
+
+def save_generated_templates(rendered_templates, generated_dir):
+    if not os.path.exists(generated_dir):
+        os.mkdir(generated_dir)
```

Comment:
The  `mkdir` create only one level of directories. How we don't know if will necessary to create one or more levels I belive that to use the `makedirs` is a better choice.
```suggestion
        os.makedirs(generated_dir)
```